homeassistant/components/input_select.py

```diff
@@ -139,3 +173,19 @@ def select_option(self, option):
             return
         self._current_option = option
         self.update_ha_state()
+
+    def select_next(self):
```

Comment:
What about having a single method with an offset for select_next and select_previous?

```python
def select_next(self, offset=1):
```

Here as a minimum, but this concept can potentially even be pulled back to the service itself